src/20231205_Multi_IncreaseSupplyAndBorrowCapsAt100UtilizationDecember2023/IncreaseSupplyAndBorrowCapsAt100UtilizationDecember2023.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Increase Supply and Borrow Caps at 100% Utilization - December 2023"
```

Comment:
```suggestion
title: "Increase Polygon wstETH Supply Cap"
```

Let's try to be specific in the title so ppl know if they want to care for the proposal or not.